### PR TITLE
Feature/UI

### PR DESCRIPTION
**Pull request by VICTOR-CHUKWU**
- [ ] created group view and controller
- [ ] Created transaction view and controller
- [ ] Used devise for authentication
- [ ] Used cancancan for authorization
- [ ] Used rspec for testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,67 @@
...
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```
Comment (bimbolabuari) [inline README.md:69]: Kindly give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) in the "Acknowledgements" section.

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @VICTOR-CHUKWU , 

You have done a good job so far 🚀  

## Highlights 🎉 

- Usage of Postgres as the database for the application.
- Usage of devise for authentication
- No linter error

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [ ] As required, Kindly record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:
  - a description of the project.
  - a demo of the project features.
  - you should also highlight some interesting piece of code or something you built that you are very proud of.
  
*For recording the video you can use tools like [Loom](https://www.loom.com/) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture.*

- [ ] As required, The project should be deployed and accessible online, Please include the deployed link to your readme file

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: README.md [hunk 2]
```diff
@@ -1,24 +1,75 @@
...
```
Comment (bimbolabuari) [inline README.md:69]: Kindly give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) in the "Acknowledgements" section.

File: README.md [hunk 3]
```diff
@@ -1,24 +1,75 @@
...
 
-* Ruby version
+## Video
+This is the link to introductory Video: [Video](https://www.loom.com/share/a315c9a60e4249038aeb5b972461e529)
 
-* System dependencies
 
-* Configuration
+## Live Link
+This is the link to heroku link: [Link](https://vic-transaction.herokuapp.com/)
```
Comment (bimbolabuari) [inline README.md:16]: As required, The deployed project should work as expected, Currently, I am getting an error with your deployed link.

image

Kindly fix this.

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @VICTOR-CHUKWU, 

Nice implementing some changes suggested by the previous reviewer 🚀  

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: README.md (hunk 3 above)
Comment (VICTOR-CHUKWU) [reply README.md:16]: - The link is working here and I have sent to other people and its working on their side too
image

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: README.md (hunk 3 above)
Comment (bimbolabuari) [reply README.md:16]: - I was referring to the sign-up route, I was unable to signup, Glad you have resolve the bug

**Pull request merged**